recipe/conda_build_config.yaml

```diff
@@ -28,7 +28,7 @@ ctng_vendor:
 #ctng_duma:
 #  - 2.5.15
 ctng_gcc_build_number:
-  - 11
+  - 12
```

Comment:
This probably doesn't need a build number bump

Reply:
Right. We don't ship the compilers very often so if we don't bump it now, we probably won't get a chance to actually ship the new license for a while.